node/coinstacks/solana/api/src/controller.ts

```diff
@@ -78,8 +91,9 @@ export class Solana implements BaseAPI {
   @Response<ValidationError>(422, 'Validation Error')
   @Response<InternalServerError>(500, 'Internal Server Error')
   @Post('send/')
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   async sendTx(@Body() _body: SendTxBody): Promise<string> {
-    return ''
+    return Solana.service.sendTx(_body)
   }
 }
+
+Solana.service = service
```

Comment:
vote to get rid of service to reduce boilerplate as I described in the comments

Reply:
Done in [412c348](https://github.com/shapeshift/unchained/pull/1049/commits/412c348ee52620a3028caadd8322dd58348bbd20)